test/e2e/x-walk/variables.spec.js

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '../fixtures.js';
+import { openPage } from '../utils.js';
+
+
+test.describe('form variables test suite', () => {
+const url = '/content/aem-boilerplate-forms-xwalk-collaterals/variables?accountCode=A1234&utm_campaign=test_campaign';
+test('check query params and browser details are getting set in properties', async ({ page }) => {
+    await openPage(page, url);
+    // on form init an api is invoked and its reponse is set as the value of text input field
```

Comment:
Where are we checking the get property use-case of query parameters

Reply:
Ignore this comment, in the test case I have added rule to set the value of the field to the passed query param